source/guides/guides/parallelization.md

```diff
@@ -68,7 +70,7 @@ As more and more tests are recorded to the Cypress Dashboard, Cypress can better
 
 ## Spec duration history analysis
 
-{% img 'no-border' /img/guides/parallelization/spec-forecast.png "Spec duration forecasting" %}
+{% img 'no-border' /img/guides/parallelization/load-balancing.png "Spec duration forecasting" %}
```

Comment:
Should be able to delete this 'spec-forecast' image if it's not being used elsewhere.